tests/README.md

```diff
@@ -3,21 +3,38 @@ xdg-desktop-portal test suite
 
 ## Environment
 
-Some relevant environment variables that can be set during testing,
-but should not normally be set on production systems:
+Some environment variables need to be set for the integration tests to function
+properly and the harness will refuse to launch if they are not set. If the
+harness is executed by meson, they will be automatically set.
 
-* `FLATPAK_BWRAP`: Path to the **bwrap**(1) executable
-    (default: discovered at build-time)
+* `XDG_DESKTOP_PORTAL_PATH`: The path to the xdg-desktop-portal binary.
+
+* `XDG_PERMISSION_STORE_PATH`: The path to the xdg-permission-store binary.
+
+* `XDG_DOCUMENT_PORTAL_PATH`: The path to the xdg-document-portal binary.
+
+* `XDP_VALIDATE_ICON`: The path to the xdg-desktop-portal-validate-icon binary.
 
-* `LIBEXECDIR`: If set, look for the x-d-p executable in this directory
+* `XDP_VALIDATE_SOUND`: The path to the xdg-desktop-portal-validate-sound binary.
```

Comment:
because english makes no sense none of the above require a `.` at the end, but the one for `XDP_VALIDATE_AUTO` does, I think...